### PR TITLE
521-Transparent navbar (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by bladebunny**
Added a modifier to allow transparent navbar on views where content should extend to the top. Such as ContentList.

#521

### CODE REVIEW

**Comment by sammyd** [comment]
@luke-freeman could you take a look at this please. Note that there are some screenshots on the linked Issue #521 

Thanks for this @bladebunny. In future, please could you add the screenshots to the PR so that it's easy for Luke to take a look and review them.

Thanks!

File: Emitron/Emitron/UI/SceneDelegate.swift [hunk 1]
```diff
@@ -41,29 +41,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
 
     // TODO: When a modifier is available these should be refactored
```
Comment (0xTim) [inline Emitron/Emitron/UI/SceneDelegate.swift:43]: If we're gutting all these lines we should remove the TODO as well

**Review by 0xTim — APPROVED** [review]
One comment nit but otherwise LGTM

File: Emitron/Emitron/UI/SceneDelegate.swift (hunk 1 above)
Comment (bladebunny) [reply Emitron/Emitron/UI/SceneDelegate.swift:43]: Currently researching a modification that allows modifying navbar transparency on scroll.  Not sure if possible but will find out.  Will amend this comment based on what i learn.

**Comment by bladebunny** [comment]
See comments I just added about requirements for a more robust implementation.  #521

**Comment by bladebunny** [comment]
Closing this PR as it doesn't quite achieve the desired behavior.  Needs to be able to progressively hide/show the navbar background on scroll.  Which is difficult to achieve with current (v 13) SwiftUI.

**Pull request closed without merging**